Initial Model - CNN.ipynb

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
<p>Can you make the output smaller? <code>X[:5]</code> is a lot of numbers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>